.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release Build
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v2
+      with:
+        java-version: 11
+        distribution: 'temurin'
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+    - name: Install gradle wrapper
+      run: gradle wrapper
+    - name: Execute Gradle build
+      run: ./gradlew build
+
+    - name: Build APK
+      run: bash ./gradlew assembleRelease
+    - name: Sign APK
+      id: sign_apk
+      uses: r0adkll/sign-android-release@v1
+      with:
+        releaseDirectory: app/build/outputs/apk/production/release/
+        signingKeyBase64: ${{ secrets.SIGNINGKEYBASE64 }}
+        alias: ${{ secrets.ALIAS }}
+        keyStorePassword: ${{ secrets.KEYSTOREPASSWORD }}
+        keyPassword: ${{ secrets.KEYPASSWORD }}
+
+    - name: Build App bundle
+      run: bash ./gradlew bundle
+    - name: Sign App Bundle
+      id: sign_aab
+      uses: r0adkll/sign-android-release@v1
+      with:
+        releaseDirectory: app/build/outputs/bundle/productionRelease/
+        signingKeyBase64: ${{ secrets.SIGNINGKEYBASE64 }}
+        alias: ${{ secrets.ALIAS }}
+        keyStorePassword: ${{ secrets.KEYSTOREPASSWORD }}
+        keyPassword: ${{ secrets.KEYPASSWORD }}
+
+
+    - name: Build Changelog
+      id: changelog
+      uses: ardalanamini/auto-changelog@v3
+
+    - name: Create release
+      uses: actions/create-release@v1
+      id: create_release
+      with:
+        draft: false
+        prerelease: false
+        release_name: Release ${{ github.ref }}
+        tag_name: ${{ github.ref }}
+        body: |
+          ${{ steps.changelog.outputs.changelog }}
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+
+    - name: Upload APK
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ${{steps.sign_apk.outputs.signedReleaseFile}}
+        asset_name: eduID-${{ github.ref_name }}.apk
+        asset_content_type: application/zip
+    - name: Upload App bundle
```

Comment:
This one is not needed, APK is enough

Reply:
The play-store prefers an app-bundle, sideloading prefers apk's; that's why we build both

Reply:
Where does the APK end up then, so that you can sideload it?
If you upload an aab to the Play Store, you can download it later as an APK from there

Reply:
The apk can be downloaded from the github release page: https://github.com/SURFnet/eduid-app-android/releases/latest